cmd.sh

```diff
@@ -55,7 +55,7 @@ if [ $POCKET_CORE_NODE_TYPE = "dispatch" ]; then
 		  --disrport ${POCKET_CORE_DISPATCH_PORT:-8081} &
 
 		sleep 10 
-		exec go test ./tests/integration/... --dispatch --dispatchtesturl dispatch.staging.pokt.network --servicetesturl service.staging.pokt.network
+		exec go test ./tests/integration/... --dispatch --dispatchtesturl ${POCKET_CORE_DISPATCH_IP:-127.0.0.1} --servicetesturl ${POCKET_CORE_DISPATCH_IP:-127.0.0.1}
```

Comment:
Shouldn't this be it's own env variable? and you used the same in both flags.

Reply:
I think we have to tests the same variables that the server is using to connect